docs/HowTo/Find-and-Connect/Specifying-NAT.md

```diff
@@ -0,0 +1,65 @@
+description: Configuring NAT with Hyperledger Besu
+<!--- END of page meta data -->
+
+# Configuring NAT 
+
+Use the [`--nat-method`](../../Reference/CLI/CLI-Syntax.md#nat-method) option to specify the 
+NAT method. Options are: `UPNP`, `MANUAL`, `DOCKER`, `AUTO`, and `NONE`. 
+
+The [enode](../../Concepts/Node-Keys.md#enode-url) advertised to other nodes during discovery is the 
+external IP address and port. The [`admin_nodeInfo`](../../Reference/API-Methods.md#admin_nodeinfo) 
+JSON-RPC API method returns the external address and port for the `enode` and `listenAddr` properties. 
```

Comment:
"... external IP address ..."?

Reply:
The enode is more than the IP address so leaving as is